packages/components/src/placeholder/style.scss

```diff
@@ -96,7 +96,7 @@
 	width: 50%;
 }
 
-.components-placeholder__fieldset .components-button {
+.block-editor-block-list__layout .components-placeholder__fieldset button.components-button {
```

Comment:
We should have this selector in addition to the previous one and not replace it because the placeholder can also be used as a generic component outside the layout.